pkg/build/registry/buildconfig/webhook.go

```diff
@@ -72,9 +76,18 @@ func (w *WebHook) ServeHTTP(writer http.ResponseWriter, req *http.Request, ctx k
 		Revision:    revision,
 		Env:         envvars,
 	}
-	if _, err := w.instantiator.Instantiate(config.Namespace, request); err != nil {
+	newBuild, err := w.instantiator.Instantiate(config.Namespace, request)
+	if err != nil {
 		return errors.NewInternalError(fmt.Errorf("could not generate a build: %v", err))
 	}
+
+	// Send back the build name so that the client can alert the user.
+	newBuildEncoded, err := runtime.Encode(kapi.Codecs.LegacyCodec(w.groupVersion), newBuild)
+	if err != nil {
+		return errors.NewInternalError(fmt.Errorf("build \"%s\" was started, but could not encode response object: %v", newBuild.Name, err))
```

Comment:
This results in seeing `server rejected our request` from the command.  You want to write the status and return 200 so a caller doesn't keep requesting builds (seeing that it's request failed), right?